sdk/ai/azure-ai-resources/azure/ai/resources/entities/connection_subtypes.py

```diff
@@ -37,7 +37,7 @@ def __init__(
         *,
         target: str,
         credentials: ApiKeyConfiguration,
-        api_version: str = "unset",
+        api_version: Optional[str] = None,
```

Comment:
is AzureOpenAIConnection a new class for 1.12.0? Otherwise, this will be a breaking change.

Reply:
Yes it's a new class. It was added in last week.